bin/hdfcoinc/pycbc_page_segplot

```diff
@@ -78,7 +84,7 @@ smin, smax = numpy.inf, -numpy.inf
 h = .7
 
 #FIXME: set IFO list
-ifos = ['H1', 'L1']
+ifos = opts.ifos
```

Comment:
remove the FIXME, since you have just fixed it!!

Reply:
Done